### DISCUSSION TITLE
Problem with error in the middle of ReVanced Builder

### DISCUSSION THREAD

**Discussion by freedom142857** [kitadai31/revanced-patches-android6-7#23] [General]
I have tried to create an APK in ReVanced Builder using https://github.com/kitadai31/revanced-patches-android6-7/wiki/How-to-build  as a reference, but an error occurs.
The source settings work fine and the original APK is https://www.apkmirror.com/apk/google-inc/youtube/youtube-17-34-36-release/youtube-17-34-36-2-android-apk-download/ I'm using
I'll post the error I got.

An error occured:Error: spawn C:\snapshot\revanced-builder\node_modules\process-exists\node_modules\ps-list\fastlist.exe ENOENTat ChildProcess._handle.onexit (node:internal/child_process:283:19)at onErrorNT (node:internal/child_process:476:16)at process.processTicksAndRejections (node:internal/process/task_queues:82:21)Please report this bug here: https://github.com/reisxd/revanced-builder/issues.

**Comment by kitadai31** [comment]
It is a known issue of ReVanced Builder.
https://github.com/reisxd/revanced-builder/issues?q=fastlist

There seems to be a workarownd.
https://github.com/reisxd/revanced-builder/issues/700#issuecomment-1527494858
He said that you can use revanced.apk in the folder.

Or, please use Revancify instead.

**Comment by freedom142857** [comment]
返信ありがとうございます。
revanced-patches-android6-7 だと手順通りやると
![スクリーンショット 2023-05-03 202555](https://user-images.githubusercontent.com/77376857/235903340-3aea5878-e6f6-4af8-b1cd-f6152388ea58.png)
しか選択できないようでした。
apkを選択して 
![スクリーンショット 2023-05-03 202923](https://user-images.githubusercontent.com/77376857/235904069-1a8f7048-dc0d-4d7d-80f9-3d86c2c9ea41.png)
1つのみ選択して
![スクリーンショット 2023-05-03 202947](https://user-images.githubusercontent.com/77
376857/235904141-2d1c734f-010b-4411-9abc-eb7883618b4a.png)
少し待つとエラーが出ました
![スクリーンショット 2023-05-03 203119](https://user-images.githubusercontent.com/77376857/235904385-ce4e345e-0f25-49a0-b9a3-a3ee80f9b14f.png)
設定も
![スクリーンショット 2023-05-03 203312](https://user-images.githubusercontent.com/77376857/235904813-7b5d9e2a-39b8-4078-b79a-070be0cb83e9.png)
間違ってなさそうでした。
ビルド初心者で間違っているところがあったら教えてください。よろしくお願いします。

**Comment by kitadai31** [comment]
ビルド自体は成功しています。
その状態で、revanced-builder-win.exeと同じ階層にある「revanced」フォルダを開いてください。
そこに「revanced.apk」があるので、それをインストールすればOKです。

これは、ReVanced Builderの3.9.1(最新バージョン)の既知の不具合です。
3.9.0を使えば発生しませんが、上の方法でrevanced.apkを取り出せばいいだけので、3.9.0に下げる必要はありません。
